ir/lib.rs

```diff
@@ -125,6 +126,11 @@ typedb_error!(
             "Relation's declared role types should not contain scopes (':').\nSource:\n{declaration}",
             declaration: typeql::statement::thing::RolePlayer
         ),
+        ModifierDefinitionError( // TODO: Should we just inline the two?
+            17,
+            "Error adding modifier",
+            source: ModifierDefinitionError
```

Comment:
1) in `typedb_error` macro if there's a source you should either use `, ( source: ModifierDefinitionError )` or `, ( typedb_error: ModifierDefinitionError )`
2) What do you mean inline?

Reply:
Inlines -> Collapse `FilterVariableNotAvailable` and `SortVariableNotAvailable` into a `ModifierVariableNotAvailable`